docs/Get started/FAQ.md

```diff
@@ -347,7 +347,7 @@ In the Fleet UI, you can turn off MDM for a host by selecting **Actions > Turn o
 
 When you turn off MDM for a host, Fleet removes the enforcement of all macOS settings for that host. Also, the host will stop receiving macOS update reminders via Nudge. Turning MDM off doesn't remove the fleetd agent from the host. To remove the fleetd agent, share [these guided instructions](#how-can-i-uninstall-the-osquery-agent) with the end user.
 
-To enforce macOS settings and send macOS update reminders, the host has to turn MDM back on. To turn MDM on, share [these guided instructions](https://fleetdm.com/docs/using-fleet/mdm-migration-guide#instructions-for-end-users) with the end user. Turning MDM back on for a host requires end user action.
```

Comment:
I cut this problem for speed.

This is because my doc changes introduce a new problem: which end user instructions do we link to in the docs? (there's now 3 sets of instructions: manual, automatic (default), automatic (end user))